rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -564,7 +570,7 @@ private Node tryToLookupNode(NodeSource nodeSource, int lookUpTimeout, String no
             logger.info("Trying to lookup a node to recover: " + nodeUrl);
             node = nodeSource.lookupNode(nodeUrl, lookUpTimeout);
         } catch (Exception e) {
-            logger.warn("Node to recover could not be looked up at URL: " + nodeUrl, e);
+            logger.warn("Node to recover could not be looked up at URL: " + nodeUrl);
```

Comment:
In this case, the catch clause is not really exceptional, because not being able to lookup a node at recovery time is likely. If we let the exception being printed, it might look like something wrong happened during execution, whereas the situation is still under control here. Do you also think now it is better to hide it in this case?